src/runtime/analytics-service-test.js

```diff
@@ -69,6 +69,7 @@ describes.realWin('AnalyticsService', {}, env => {
   describe('AnalyticsService', () => {
     it('should have analyticsService constructed', () => {
       const activityIframe = analyticsService.getElement();
+      expect(analyticsService.getTransactionId()).to.not.undefined;
```

Comment:
But it's never `undefined` right now...

Reply:
yes, just to ensure that it is NOT `undefined`

Reply:
Maybe let's replace this with a match of some sort? E.g. `expect(getTransactionId()).to.match(/ddd-ddd-ddd..../)`?